content-src/components/DiscoveryStreamComponents/DSLinkMenu/DSLinkMenu.jsx

```diff
@@ -43,18 +43,15 @@ export class _DSLinkMenu extends React.PureComponent {
     const {index, dispatch} = this.props;
     const isContextMenuOpen = this.state.showContextMenu && this.state.activeCard === index;
     const TOP_STORIES_CONTEXT_MENU_OPTIONS = ["CheckBookmarkOrArchive", "CheckSavedToPocket", "Separator", "OpenInNewWindow", "OpenInPrivateWindow", "Separator", "BlockUrl"];
-    const title = this.props.title || this.props.source;
     const type = this.props.type || "DISCOVERY_STREAM";
+    const title = this.props.title; // eslint-disable-line prefer-destructuring
```

Comment:
How did you write the destructuring? Although, why not just keep the original `title || source` ? But then again, seems unlikely the title will require a fallback…

Reply:
You know, I don't know! I'm not sure when I removed the `title || source` but I don't see why it wouldn't work. I'll try switching it back.

Reply:
That seems to work fine and the tests pass, so that's that!